airflow/providers/amazon/aws/hooks/eventbridge.py

```diff
@@ -16,12 +16,76 @@
 # under the License.
 from __future__ import annotations
 
+import json
+
+from airflow import AirflowException
 from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+from airflow.providers.amazon.aws.utils import trim_none_values
+
+
+def validate_json(pattern: str) -> None:
+    try:
+        json.loads(pattern)
+    except ValueError:
+        raise AirflowException("Event pattern must be a valid JSON string.")
 
 
 class EventBridgeHook(AwsBaseHook):
     """Amazon EventBridge Hook."""
 
     def __init__(self, *args, **kwargs):
-        """Creating object."""
         super().__init__(client_type="events", *args, **kwargs)
+
+    def put_rule(
+        self,
+        name: str,
+        description: str | None = None,
+        event_bus_name: str | None = None,
+        event_pattern: str | None = None,
+        role_arn: str | None = None,
+        schedule_expression: str | None = None,
+        state: str | None = None,
+        tags: list[dict] | None = None,
+        **kwargs,
+    ):
+        """
+        Create or update an EventBridge rule.
+
+        :param name: name of the rule to create or update (required)
+        :param description: description of the rule
+        :param event_bus_name: name or ARN of the event bus to associate with this rule
+        :param event_pattern: pattern of events to be matched to this rule
+        :param role_arn: the Amazon Resource Name of the IAM role associated with the rule
+        :param schedule_expression: the scheduling expression (for example, a cron or rate expression)
+        :param state: indicates whether rule is set to be "ENABLED" or "DISABLED"
+        :param tags: list of key-value pairs to associate with the rule
+
+        """
+        if not (event_pattern or schedule_expression):
+            raise AirflowException(
```

Comment:
I think `ValueError` makes more sense here

```suggestion
            raise ValueError(
```